installer.py

```diff
@@ -0,0 +1,293 @@
+
+
+def abort():
+    """Aborts the installation process and deletes the '.env' file."""
+    os.remove(".env")
+    print(TRED + "\nInstallation process aborted. 🫠", TWHITE)
+    sys.exit()
+
+
+def main():
+    """Run main function to begin the installation process."""
+    print("Welcome to the Talawa API installer! 🚀")
+
+    env_creation()
+    access_and_refresh_tokens()
+
+    try:
+        choice = int(
+            input(
+                "\nWould you like to use a local instance or a cloud instance of MongoDB?\n\n0. Local\n1. Cloud\n\nEnter choice: "
+            )
+        )
+
+        if choice == 0:
+            url = "mongodb://localhost:27017/talawa-api?retryWrites=true&w=majority"
+            success = check_connection(url)
+            if success:
+                with open(".env", "r", encoding="utf-8") as file:
+                    lines = file.readlines()
+                replace_line(
+                    ".env",
+                    [i for i, line in enumerate(lines) if "MONGO_DB_URL=" in line][0],
+                    "MONGO_DB_URL=mongodb://localhost:27017/talawa-api?retryWrites=true&w=majority\n",
+                )
+                db_url = url
+            else:
+                print(
+                    TRED + "\nConnection to MongoDB failed. Please try again.", TWHITE
+                )
+                print(TRED + "\nTry starting up MongoDB on your local machine", TWHITE)
+                abort()
+        elif choice == 1:
+            cloud_instance = input("\nEnter your MongoDB cloud instance URL: ")
+            success = check_connection(cloud_instance)
+            if success:
+                with open(".env", "r", encoding="utf-8") as file:
+                    lines = file.readlines()
+                replace_line(
+                    ".env",
+                    [i for i, line in enumerate(lines) if "MONGO_DB_URL=" in line][0],
+                    f"MONGO_DB_URL={cloud_instance}\n",
+                )
+                db_url = cloud_instance
+            else:
+                print(
+                    TRED + "\nConnection to MongoDB failed. Please try again.", TWHITE
+                )
+                print(TRED + "\nTry starting up MongoDB on your local machine", TWHITE)
+                abort()
+            db_url = cloud_instance
+        else:
+            print("\nInvalid choice. Please try again.\n")
+            abort()
+    except ValueError:
+        print("\nInvalid choice. Please try again.\n")
+        abort()
+
+    recaptcha()
+    two_factor_auth()
+
+    client = MongoClient(db_url)
+    db = client["talawa-api"]
+    users_collection = db["users"]
+
+    new_user = {
+        "firstName": "test",
+        "lastName": "admin acc",
+        "email": "test@email.com",
+        "password": "Pass@1234",
+        "userType": "SUPERADMIN",
+        "tokenVersion": 0,
+        "appLanguageCode": "en",
+        "createdOrganizations": [],
+        "createdEvents": [],
```

Comment:
1. Does this test whether the user already exists?
2. Does this test whether the user already has the privileges?

Reply:
Well as of now it doesn’t check anything of that sort exists in the DB.
I'll add a check for that, and if a SUPERADMIN account already exists I'll just display its credentials.

Reply:
@palisadoes if a ```SUPERADMIN``` already exists, what should be the flow of the script?
Should I ask the user if he would like to create a new ```SUPERADMIN``` or not create a new ```SUPERADMIN``` bat all?

Reply:
It should just mention that one is already found.

Reply:
okay